packages/data-lake-api/src/DataLakeAnalyticsFetchQuery.js

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
Can we have this be a typescript package please :) JS is so 1990s

Reply:
Hahaha I'm with ya 😂 But I was kinda hoping to get this PoC merged in (with all it's code issues) and then fix them up shortly in [RN-242](https://linear.app/bes/issue/RN-242/add-data-lake-data-service-in-tupaia)? 

More than just switching to typescript, I also wanna refactor out common code into utils packages, and take a bit more time to consider how the api should work.

The reason for merging this in its current state would just be to allow Juliana to start working with it, as I don't think any changes I'd make would impact her side of things 👍

Reply:
Ha I get it, I think it will be relatively quick though no? Especially since right now we are pretty loose on anys